Customer.io project/components/Header.tsx

```
@@ -85,47 +91,49 @@ export const Header: React.FC<HeaderProps> = ({
           >
             {isSidebarOpen ? <X size={20} /> : <Menu size={20} />}
           </Button>
 
           {/* Subject and From fields */}
-          <div className="hidden lg:flex  flex-col sm:flex-row gap-4">
+          <div className="hidden lg:flex flex-col sm:flex-row gap-4">
             <div className="flex flex-col space-y-1">
-              <label className="text-xs font-medium text-gray-700">
+              <label className="text-xs font-medium text-gray-700 dark:text-white/70">
                 Subject
               </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)